src/mesh/mesh_visualization.jl

```diff
@@ -68,3 +68,48 @@ RecipesBase.@recipe function f(m::VertexMeshPlotter{2})
     return xmesh, ymesh
 end
 
+"""
+    Meshdata_to_vtk(md, rd, dim, data, dataname, datatype, filename, write_data = false, equi_dist_nodes = true)
+
+Translate the given mesh into a vtk-file.
+md holds the meshdata
+rd holds the reference element data. 
+dim is the dimension of the mesh
+data holds an array of arryas with data
+dataname is an array of strings with name of the associated data
+write_data, flag if data should be written or not
+equi_dist_nodes flag if points should be interpolated to equidstant nodes
+"""
+function Meshdata_to_vtk(md::MeshData, rd::RefElemData, dim, data, dataname, filename, write_data = false, equi_dist_nodes = true)
+    # Compute the permutation between the StartUpDG order of points and vtk
+    perm = SUD_to_vtk_order(rd, dim)
+    # The number of points per element
+    num_lagrange_points = size(perm)[1]
```

Comment:
```suggestion
    num_lagrange_points = size(perm, 1)
```
Or if `perm` is a vector, just doing `length` is clearer.